src/index.css

```diff
@@ -262,6 +276,7 @@ h1 {
     padding: 20px 40px;
     width: 40%;
     margin: 30px 30px;
+    border-radius:20px;
 }
```

Comment:
We could also add a background-color of white to the .details-container to make it more distinct from the background?

Reply:
Hi Allison, Will be add background-color of white to .details-container.